test/Tests/Old.hs

```diff
@@ -99,6 +99,10 @@ tests = [ testGroup "markdown"
           , test "reader" ["-r", "mediawiki", "-w", "native", "-s"]
             "mediawiki-reader.wiki" "mediawiki-reader.native"
           ]
+        , testGroup "vimwiki"
+          [ test "reader" ["-r", "vimwiki", "-w", "html5", "-s"]
```

Comment:
The test output should be "native", not "html5" (that way we only test the reader, not also the HTML writer).
See how it's done above with e.g. mediawiki.